qaframework-bdd-tests/src/test/resources/features/refapp-2.x/10-allergy/allergies.feature

```diff
@@ -1,17 +1,31 @@
 Feature: Allergies Management
 
   Background:
-    Given a user clicks on Allergies link from Patient dashboard
-    Then the system loads Allergies page
+    Given a user clicks on Allergies link from the Patient dashboard
+    Then the system loads the Allergies board page
 
   @selenium
-  @dashboard
-  Scenario: Add No known allergies
-    And a user clicks No Known Allergy button
-    Then the system add no known allergies into the allergies table
-
-  @selenium
-  @dashboard
-  Scenario: Remove No known allergies
-    And a user clicks Remove No Known Allergy icon
+  @allergies
+  Scenario: Allergies management
+    # User story: Add No known allergies
+    When a user clicks No Known Allergy button
+    Then the system adds no known allergies into the allergies table
+    # User story: Remove No known allergies
+    When a user clicks Remove No Known Allergy icon
     Then the system displays unknown in the allergies table
+    # User story: Add known allergy
```

Comment:
Hey @Irenyak1 .This looks very awesome. Was suggesting that you separate these scenarios in to like three different scenario test cases to make it more readable.

Reply:
> Hey @Irenyak1 .This looks very awesome. Was suggesting that you separate these scenarios in to like three different scenario test cases to make it more readable.

@sherrif10  if I separate them into scenarios won't the different scenarios be dealing with a different patient yet I would think that they should deal with the same patient. Or maybe I could separate them into two scenarios, the first one dealing with no known allergies and the second dealing with known allergies. How do you see?

Reply:
@sherrif10 I await your feedback so I can proceed

Reply:
@sherrif10 will you respond to this when you get time please.

Reply:
Separating scenarios doesnot cause the workflow failure or pass. However i would consider this approved. cc @kdaud . your final reviews please and get this enter the master.